node/10-user/Dockerfile

```diff
@@ -28,3 +28,19 @@ ENV PATH="/home/node/.pyenv/bin:/home/node/.pyenv/shims:${PATH}"
 
 RUN pyenv install 3.7.1
 RUN pyenv global 3.7.1
+
+# Setup Cloud SDK
+WORKDIR /home/node
```

Comment:
will `WORKDIR` have an effect on the default entry point for the container? I think currently it's a assumed to be `/usr/local/bin`:

https://github.com/nodejs/docker-node/blob/f8f2384f7edc345f5ffc0496458005981b512882/10/stretch/Dockerfile#L64

Is there an easy way to ensure that we leave `WORKDIR` the default value.

Reply:
WORKDIR does indeed set the default entry working directory, but that COPY statement does not. The WORKING directory before this change was `/`, I can add another WORKDIR at the end to set it back to that value.